packages/fuels-test-helpers/src/node.rs

```diff
@@ -264,10 +265,15 @@ pub async fn new_fuel_node(
             args.push("--manual_blocks_enabled");
         }
 
-        let mut running_node = Command::new("fuel-core").args(args)
+        let mut command = Command::new("fuel-core");
+        if silent {
+            command
+                .stdin(Stdio::null())
```

Comment:
Stdin should always be closed since we're not aiming to have an interactive experience with the spawned fuel-core process.

Reply:
^ Good point!